js/src/hooks/useRaiseBudgetRecommendations.test.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * External dependencies
+ */
+import { renderHook } from '@testing-library/react';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import useRaiseBudgetRecommendations from './useRaiseBudgetRecommendations';
+import useGoogleAdsAccount from './useGoogleAdsAccount';
+
+jest.mock( '@wordpress/data/src/components/use-select', () => jest.fn() );
+
+jest.mock( '@wordpress/data', () => ( {
+	__esModule: true,
+	useSelect: jest.fn(),
+} ) );
+
+jest.mock( './useGoogleAdsAccount', () =>
+	jest.fn().mockName( 'useGoogleAdsAccount' )
+);
+
+const mockedRecommendations = [
```

Comment:
@asvinb just noting that these types of recommendation will have a `details` key with the budget options inside. I don't think it's required for the mocks created for these test so this is just an FYI